src/ComputeSharp.D2D1/Attributes/Enums/D2D1ChannelDepth.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ComputeSharp.D2D1;
+
+/// <summary>
+/// Indicates the channel depth of a stage in a Direct2D rendering pipeline.
+/// </summary>
+/// <remarks>
+/// <para>This type exposes the available values in <see href="https://docs.microsoft.com/windows/win32/api/d2d1effectauthor/ne-d2d1effectauthor-d2d1_channel_depth"/>.</para>
+/// <para>For more info, see <see href="https://docs.microsoft.com/windows/win32/direct2d/precision-and-clipping-in-effect-graphs"/>.</para>
+/// </remarks>
+public enum D2D1ChannelDepth
+{
+    /// <summary>
+    /// The channel depth is the default, and it will be inherited by the inputs.
```

Comment:
inherited _from_ the inputs? https://docs.microsoft.com/en-us/windows/win32/api/d2d1effectauthor/ne-d2d1effectauthor-d2d1_channel_depth

<img width="313" alt="image" src="https://user-images.githubusercontent.com/10873410/167271753-b4ebc4f0-636d-4301-8e10-d2daa3d1e4d5.png">

Reply:
Fixed in e81a39d27aced6e052ab8424d8fa9622a88d683e.